src/style.css

```diff
@@ -1,3 +1,68 @@
-body {
+:root {
+  --current-text: #000;
+  --brend-color: #00acdc;
+}
+
+.body {
+  margin: 0;
+  font-family: "Roboto", "Arial", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
   margin: 0;
+  padding: 0;
+}
+
+a {
```

Comment:
use classes instead of tag selector

Reply:
I cant add class to tag "a" because   "29:48   error  Only 2 attributes per line are permitted  attr-new-line "

Reply:
It "Laptops & computers" has 3 atribute

Reply:
please check